domain/consensus/processes/coinbasemanager/coinbasemanager.go

```diff
@@ -141,6 +142,14 @@ func (c *coinbaseManager) coinbaseOutputForRewardFromRedBlocks(stagingArea *mode
 	}, true, nil
 }
 
+func acceptanceDataFromArrayToMap(acceptanceData externalapi.AcceptanceData) map[externalapi.DomainHash]*externalapi.BlockAcceptanceData {
+	acceptanceDataMap := make(map[externalapi.DomainHash]*externalapi.BlockAcceptanceData)
```

Comment:
You can convert it to `make(map[externalapi.DomainHash]*externalapi.BlockAcceptanceData, len(acceptanceData))` to preallocate the map

Reply:
Great, thanks.